RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -273,9 +275,24 @@ void sweatpantsPreAdventure() {
 	}
 }
 
+boolean auto_hasStillSuit()
+{
+	return possessEquipment($item[tiny stillsuit]) && auto_is_valid($item[tiny stillsuit]);
+}
+
+int auto_expectedStillsuitAdvs()
+{
+	if(!auto_hasStillSuit()) return 0;
+	int sweat = get_property("familiarSweat").to_int();
+	// can't consume until at least 10 sweat has been accumulated
+	if(sweat < 10) return 0;
+
+	return(round(sweat**0.4));
+}
+
 void utilizeStillsuit() {
 	//called at the end of pre adv to make sure stillsuit is at least kept equipped on a familiar in the terrarium
-	if(item_amount($item[tiny stillsuit]) == 0)
+	if(!auto_hasStillSuit())
```

Comment:
previous check was for unequipped stillsuit in inventory rather than possessing one, but there is already an explicit check that it's not equipped afterwards so you could just remove the comments that say
>since it's in the inventory

Reply:
Done